x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/translations.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ENABLE_ENTITY_STORE_TITLE = i18n.translate(
+  'xpack.securitySolution.entityAnalytics.entityStore.enablement.title.store',
+  {
+    defaultMessage: 'Enable Entity Store',
+  }
+);
+export const ENABLE_RISK_SCORE_TITLE = i18n.translate(
+  'xpack.securitySolution.entityAnalytics.entityStore.enablement.title.risk',
+  {
+    defaultMessage: 'Enable Risk Score',
```

Comment:
```suggestion
    defaultMessage: 'Enable entity risk score',
```

Reply:
![image](https://github.com/user-attachments/assets/db5ce168-ca71-46c7-a2c1-c571d6ec145b)

If this UI is for enabling risk scores only (when entity store is already enabled), why does it have the same description as for enabling entity store?

Reply:
Sorry, added another question here.

Reply:
oh! Good point! I think that was an oversight.

Reply:
yeah, the text should be: 

Risk engine only:
> Provides real-time visibility into user activity, helping you identify and mitigate potential security risks.

Entity store only:
> Allows comprehensive monitoring of your system's hosts and users. 

Both: the current text

This is the same text as in the modal

Reply:
Updated!